### PR TITLE
Create 2021-08-20-micro-acquisition-pilot-update2.md

### PR DESCRIPTION
**Pull request by RachelMuston**

### CODE REVIEW

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+layout: post
+title:  "Micro-Acquisition Pilot: Status Update 2"
+ref: micro-acquisition-update2
+lang: en
+author: "Rachel Muston, in collaboration with the IT Strategy team and Micro-Acquisition Pilot team members"
+date: "2021-08-20"
+excerpt: "The Micro-Acquisition pilot is live! In this blog post we share what we have learned so far and how we are adjusting and improving."
+---
+<!--markdownlint-disable MD033-->
+Since our last status update, the [Micro-Acquisition website](https://micro-acquisition.alpha.canada.ca/en/index.html) went live and the [first opportunity](https://micro-acquisition.alpha.canada.ca/2021/06/23/building-custom-Azure-marketplace-policies.html) was posted.
```
Comment (gcharest) [inline _posts/en/2021-08-20-micro-acquisition-pilot-update2.md:11]: Should we move links to the footnote as per the guidelines you showed us before to avoid click bait?

I was thinking about moving the links out of the paragraph and put them in a blockquote or as a footnote(1)?

> [Micro-Acquisition website](https://micro-acquisition.alpha.canada.ca/en/index.html)

(1): [Micro-Acquisition website](https://micro-acquisition.alpha.canada.ca/en/index.html)

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Added a mailing list
+
+When we didn't get a lot of followers on our [Twitter account](https://twitter.com/microbuysgc), we did some testing and realized 2 things:
+
+1. It wasn't clear what information would be found by following the Twitter account and
+2. A lot of our target audience is not on Twitter
+
+We now have a mailing list that anyone can subscribe to.
+Through the [mailing list](https://forms-formulaires.alpha.canada.ca/id/36) (and Twitter) we will be sharing when new opportunities are posted and/or updated. We also updated the text on the opportunities page to be more clear what information is available via the Twitter account and mailing list.
```
Comment (gcharest) [inline _posts/en/2021-08-20-micro-acquisition-pilot-update2.md:24]: > We also updated the text on the opportunities page to be more clear _[about]_ what information is available via the Twitter account and mailing list.

I'm wondering if there shouldn't be a "about" in the sentence aboce?

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Updated evaluation question
+
+Our first opportunity was for a less well known language (Powershell).
```
Comment (gcharest) [inline _posts/en/2021-08-20-micro-acquisition-pilot-update2.md:28]: Was Powershell the cause for people writing answers that didn't pass or is it our hypothesis that the less popular nature of the language may have been a factor in receiving less applications than expected?

If it's a possible factor in less applications than expected, maybe this should be in a different section: Communicating/Advertising to target communities or something like that?

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+We realized from the applications we received that there were people who likely could do the work but the phrasing of our evaluation criteria prevented us from giving them a pass because they didn't have experience specifically in Powershell.
```
Comment (gcharest) [inline _posts/en/2021-08-20-micro-acquisition-pilot-update2.md:29]: If the previous note about Powershell should be in a different section, then the focus here is less about Powershell and the way applicants framed their answers, as per the rest of this section.

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+We've also added [example answers which would get a passing mark](https://micro-acquisition.alpha.canada.ca/en/user-guide.html#example-answers-that-would-get-a-passing-mark) to the User Guide.
+
+## Using GC Forms for the application form
+
+We are replacing our pdf application form with a form on [GC Forms](https://forms-formulaires.alpha.canada.ca/en/welcome-bienvenue).
+GC Forms is a CDS product and it is accessible, secure and has full authority to operate.
+GC Forms is a service - you tell CDS your need, they build your form, test it and then send you the link.
+
+IT IS GREAT.
```
Comment (gcharest) [inline _posts/en/2021-08-20-micro-acquisition-pilot-update2.md:45]: Why is it great?

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+We are excited because not only will we no longer have to maintain a pdf form, but user experience for suppliers will be improved as the form will tell them if they have missed filling something in.
```
Comment (gcharest) [inline _posts/en/2021-08-20-micro-acquisition-pilot-update2.md:47]: I think mentioning the Accessibility, Privacy and Security could good  as well :)

**Review by gcharest — COMMENTED** [review]
First review  with a few thoughts :smile: 

Great read and not too long :+1:

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md (hunk 1 above)
Comment (gcharest) [reply _posts/en/2021-08-20-micro-acquisition-pilot-update2.md:11]: Blockquotes require the use of ">" in front of paragraphs.

Footnotes use the following link model `[^1]` in the paragraph and then `[^1]: ` at the bottom of the page.

Comment (gcharest) [reply _posts/en/2021-08-20-micro-acquisition-pilot-update2.md:11]: Link to guide: https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-style-guide.html#toc11

Comment (gabrielcossette) [reply _posts/en/2021-08-20-micro-acquisition-pilot-update2.md:11]: Maybe some excitement could be added, like a "yeah!" at the end of the sentence?

**Review by gabrielcossette — APPROVED** [review]
Great work, a short good read!

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md (hunk 1 above)
Comment (RachelMuston) [reply _posts/en/2021-08-20-micro-acquisition-pilot-update2.md]: Good point @gcharest! I've changed three of the links to footnotes and only kept the links that are related to the changes we talk about in the blog post. What do you think?

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md (hunk 2 above)
Comment (RachelMuston) [reply _posts/en/2021-08-20-micro-acquisition-pilot-update2.md]: Yes! Fixed. Good catch :)

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md (hunk 3 above)
Comment (RachelMuston) [reply _posts/en/2021-08-20-micro-acquisition-pilot-update2.md]: I think it was a bit the cause of people writing answers that didn't pass because I think for some folks it was like, I know I don't have that skill but I'm interested and I'll try anyway.  But even still, after re-reading I think that first sentence didn't add to the understanding of this item, so I removed it. :)

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md (hunk 5 above)
Comment (RachelMuston) [reply _posts/en/2021-08-20-micro-acquisition-pilot-update2.md]: Good point. I've reworked this paragraph with some language from the GC Forms website. Let me know what you think. I'll probably also run this text by CDS.

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md (hunk 6 above)
Comment (RachelMuston) [reply _posts/en/2021-08-20-micro-acquisition-pilot-update2.md]: I think I've got this in the reworked text but let me know if you think I could/should add more. :)

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md (hunk 3 above)
Comment (philipe-girard) [reply _posts/en/2021-08-20-micro-acquisition-pilot-update2.md]: I wouldn't say that Powershell is a less popular language.  In fact it is used alot in the world of infrastructure.  I would reword to state that it is a "niche" language.

Comment (RachelMuston) [reply _posts/en/2021-08-20-micro-acquisition-pilot-update2.md]: Oh I actually took that line out after Guillaume's review as I don't think it added to the point re: the updated evaluation question. So we are all good there.

File: _posts/en/2021-08-20-micro-acquisition-pilot-update2.md (hunk 6 above)
Comment (RachelMuston) [reply _posts/en/2021-08-20-micro-acquisition-pilot-update2.md]: This version has been sent to translation. Will be back on Sept. 13.

**Comment by gcharest** [comment]
Waiting on translation for publishing, am I correct? @RachelMuston @gabrielcossette

**Review by gcharest — CHANGES_REQUESTED** [review]
Need translation + review

**Comment by RachelMuston** [comment]
Yes thanks for adding that!  Also waiting for 'the fall launch'.  It's due to be posted on or around Oct. 6.  https://014gc.sharepoint.com/:w:/r/sites/GCDevEx/Shared%20Documents/Change%20Management/Comms/Full%20launch_comms_calendar.docx?d=we98b49c44fed42fb919951548601bcc5&csf=1&web=1&e=8zXIJo

**Comment by RachelMuston** [comment]
Almost forgot...this blog post is waiting on a few other things before posting:
1. CFOB review (cassandra has it and is taking it to Robert for his review)
2. GC Forms team is validating the text about their service.

**Comment by gabrielcossette** [comment]
FR translation added and reviewed. 👍

I also removed the references to Acrobat and "export to PDF from .odt" in both versions . I thought it would be better to not mention product names when not required (we still understand the point).
